packages/vanilla/src/components/global-nav/top-nav/notifications/notifications.js

```diff
@@ -38,6 +38,7 @@ class Notifications extends Core {
     this.flyout.addSlot(this.list);
     this.flyout.setAnchorPoint('top-right');
     window.addEventListener('resize', this._adjustFlyoutMaxHeight.bind(this));
+    window.addEventListener('scroll', this._adjustFlyoutMaxHeight.bind(this));
```

Comment:
It seems in the spirit of the design to also adjust the height when scrolling the browser window. This is demonstrated in the PR screenshot.